plugins/generator-1.17.1/forge-1.17.1/templates/block/block.java.ftl

```diff
@@ -70,7 +70,7 @@ public class ${name}Block extends
 	</#if>
 
 	<#macro blockProperties>
-		BlockBehaviour.Properties.of(Material.${data.material})
+		BlockBehaviour.Properties.of(Material.${data.material}<#if generator.map(data.colorOnMap, "mapcolors") != "DEFAULT">,MaterialColor.${generator.map(data.colorOnMap, "mapcolors")}</#if>)

```

Comment:
Plants have multiline if here, make it so here too as it is more readable:

![image](https://user-images.githubusercontent.com/16374228/150674270-1178d42e-e13a-452c-9322-0ed629a9dd8f.png)